src/services/services.ts

```diff
@@ -6740,7 +6741,7 @@ namespace ts {
                 fileName: node.getSourceFile().fileName,
                 textSpan: createTextSpanFromBounds(start, end),
                 isWriteAccess: isWriteAccess(node),
-                isDefinition: isDeclarationName(node)
+                isDefinition: isDeclarationName(node) || node.parent.kind === SyntaxKind.ComputedPropertyName

```

Comment:
it's not true all the time -- isDeclarationName additionally checks that `node.parent.name === node` which needs to be generally true for something to be a valid declaration. Computed property names don't meet this requirement because often we can't calculate the name at compile time. In this code path, however, we know that the computed property name already, so I added the special case here only .